packages/hint-scoped-svg-styles/package.json

```
@@ -0,0 +1,77 @@
+{
+  "ava": {
+    "failFast": false,
+    "files": [
+      "dist/tests/**/*.js"
+    ],
+    "timeout": "1m"
+  },
+  "description": "Scoped SVG Styles checks if SVG styles affect any other elements outside the svg.",
+  "dependencies": {
+    "@hint/utils": "^3.1.1"
```

This should fix your `HTMLElement` issue (I see you already updated `hint`). Make sure you re-run `yarn` locally after updating so it removes any older version it may have previously installed. 
```suggestion
    "@hint/utils": "^4.0.0"
```

For context, we published a number of new package versions to `npm` yesterday which is why this got out of sync. Adding a new hint is one of the few times we get bit by this since we auto-update the `package.json` for any existing hints as part of the release process. Even then it probably wouldn't have affected you if we hadn't been modifying `@hint/utils` in addition to adding your new hint. Sorry for the confusion.